GoogleForCreators/web-stories-wp
# Text presets: smart color

## Context



## Summary
This PR ensures that any text element (including presets) added to the page has good contrast with its background.
Uses either black/white colour or semi-transparent black/white Fill (the exact fill colour to be adjusted)


## Relevant Technical Choices



## To-do

- [x] Update the fill colour based on Figma



## User-facing changes



## Testing Instructions
1. Set the background colour to black
2. Add a text (either from the `+` icon or any preset)
3. Verify the text is added so that it's visible from the background (assumably white on the black background)
4. Test with different backgrounds and different presets
5. Test with using Zoom
6. Verify the text is always added so that it's accessible and the contrast between the text background and text colour is accessibly.


### QA
See above.





This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #7986
